docs/integrate-logto/interact-with-management-api/README.md

```diff
@@ -0,0 +1,144 @@
+---
+description: This section covers how to interact with Logto Management API.
+sidebar_position: 2
+---
+
+import logtoManagementApiResourceSrc from './assets/logto-management-api-resource.png';
+import logtoManagementApiDetailsSrc from './assets/logto-management-api-details.png';
+
+import BasicsAboutAccessTokenRequest from '../../quick-starts/generic/machine-to-machine/fragments/\_basics-about-access-token-request.mdx';
+import FetchAccessTokenForLogtoManagementApi from '../../quick-starts/generic/machine-to-machine/fragments/\_fetch-access-token-for-logto-management-api.mdx';
+import AccessTokenUsage from '../../quick-starts/generic/machine-to-machine/fragments/\_access-token-usage.mdx';
+import AccessLogtoManagementApiUsingAccessToken from '../../quick-starts/generic/machine-to-machine/fragments/\_access-logto-management-api-using-access-token.mdx';
+import M2mAccessTokenNote from '../../quick-starts/generic/machine-to-machine/fragments/\_m2m-access-token-sub-note.mdx';
+import M2mRoleAssignment from '../../quick-starts/generic/machine-to-machine/fragments/\_m2m-role-assignment.mdx';
+
+# Interact with Management API
+
+## What is Logto Management API
+
+The Logto Management API is a comprehensive set of APIs that gives developers full control over their implementation to suit their product needs and tech stack. It is pre-built, listed in the API resource list in the Logto Console, and cannot be deleted or modified.
+
+Its identifier is in the pattern of `https://[tenant-id].logto.app/api`
+
+<img alt="Logto Management API Resource" src={logtoManagementApiResourceSrc} />
+
+<img alt="Logto Management API Details" src={logtoManagementApiDetailsSrc} />
+
+With the Logto Management API, you can access Logto's robust backend services, which are highly scalable and can be utilized in a multitude of scenarios. It goes beyond what's possible with the Admin Console's low-code capabilities.
+
+Some frequently used APIs are listed below:
+
+- User
+- Application
+- Logs
+- Roles
+- Resources
+- Connectors
+
+To learn more about the APIs that are available, please visit https://openapi.logto.io/.
+
+## How to access Logto Management API
+
+### Create an M2M app
+
+On the "Applications" page, select the M2M app type and start the creation process.
```

Comment:
Use `<CloudLink to"/applications">Console > Applications</CloudLink>` instead?

Reply:
Updated.